### PR TITLE
use go install instead of go get

### PR DESCRIPTION
**Pull request by shogo82148**
Install binaries by `go get` is deprecated.
We should use `go install` instead it.

### CODE REVIEW

**Comment by itchyny** [comment]
Can you remove the line setting `GO111MODULE` in Makefile?

**Comment by shogo82148** [comment]
sure. I removed it.

**Comment by Songmu** [comment]
Thank you for all the maintenance. as for linter, I prefer to use staticcheck as a stand-alone item, what do you think? @shogo82148

**Comment by shogo82148** [comment]
I agree with you. Let's migrate to staticcheck.

File: Makefile [hunk 1]
```diff
@@ -4,31 +4,24 @@ BUILD_LDFLAGS = "-s -w -X main.revision=$(CURRENT_REVISION)"
...
+	go install honnef.co/go/tools/cmd/staticcheck@latest
 
 .PHONY: test
 test: deps
 	go test $(VERBOSE_FLAG) ./...
 
 .PHONY: lint
 lint: devel-deps
-	golint -set_exit_status ./...
+	golangci-lint run
```
Comment (itchyny) [inline Makefile:24]: Should be `staticcheck ./...`?

Comment (Songmu) [reply Makefile:24]: I fixed it with another commit.

**Pull request merged**